docs/src/pages/components/text-fields/TextFieldSizes.tsx

```diff
@@ -1,23 +1,17 @@
 import * as React from 'react';
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      '& .MuiTextField-root': {
-        margin: theme.spacing(1),
-        width: 200,
-      },
-    },
-  }),
-);
-
 export default function TextFieldSizes() {
-  const classes = useStyles();
-
   return (
-    <form className={classes.root} noValidate autoComplete="off">
+    <Box
+      component="form"
+      sx={{
+        '& .MuiTextField-root': { m: 1, width: '25ch' },
```

Comment:
```suggestion
        '& .MuiTextField-root': { m: 1, width: 200 },
```
?

Reply:
As explained [here](https://github.com/mui-org/material-ui/pull/25626#discussion_r608010702) we leave it with `ch`?